### ISSUE TITLE
Bad `CREATE MATERIALIZED VIEW` will cause FE down

### ISSUE THREAD

**Issue by padmejin** [StarRocks/starrocks#10347]
### Steps to reproduce the behavior (Required)
```
mysql> create table customer (c_name varchar(100), c_phone int(18)) DUPLICATE KEY(c_name) DISTRIBUTED BY HASH(c_name) BUCKETs 4 PROPERTIES("replication_num" = "1");
Query OK, 0 rows affected (0.02 sec)

mysql> CREATE MATERIALIZED VIEW mv_customer_sumc_phone AS
    -> EXPLAIN SELECT c_name, MAX(c_phone)
    -> FROM customer
    -> GROUP BY c_name;
Query OK, 0 rows affected (0.06 sec)
```

### Expected behavior (Required)
Execute failed with error messages.

### Real behavior (Required)
It seems to execute well, but later all FE are down when replaying the journal.

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`
2.3.0

**Comment by zhanqian-1993** [comment]
@padmejin  hello, I want to know how to reboot FE successfully?  I have the same problem

**Comment by padmejin** [comment]
We'll release 2.3.2 later to fix this. You can manually merge the following 2 PRs, then build a new FE and replace the current one. Please leave a note if you have any problem building FE.
https://github.com/StarRocks/starrocks/pull/10360
https://github.com/StarRocks/starrocks/pull/10456

**Comment by padmejin** [comment]
> @padmejin hello, I want to know how to reboot FE successfully? I have the same problem

We have  just released 2.3.2 https://github.com/StarRocks/starrocks/releases/tag/2.3.2. Please upgrade to this version if you still have this problem ~
